pkg/bpf/metrics.go

```diff
@@ -3,10 +3,63 @@
 
 package bpf
 
+import (
+	"context"
+
+	"github.com/cilium/hive/job"
+	"github.com/cilium/statedb"
+
+	"github.com/cilium/cilium/pkg/metrics"
+	"github.com/cilium/cilium/pkg/time"
+)
+
 const (
 	metricOpCreate     = "create"
 	metricOpUpdate     = "update"
 	metricOpLookup     = "lookup"
 	metricOpDelete     = "delete"
 	metricOpGetNextKey = "getNextKey"
 )
+
+const (
+	tablePressureMetricsInterval = 30 * time.Second // Interval for updating the pressure gauge
```

Comment:
Great question to ask in general. Here my reasoning is that the pressure metric is a slow indicator for the map pressure so its exact value doesn't matter as long as it's "reasonable" at a human time scale. It's probably not pulled very often either. I can't imagine a need for actually configuring this to be a lower value in production, except maybe for demo purposes? My preference would be to avoid making things configurable that clearly don't need to be as it increases the overall complexity of the software.